src/components/Button/Button.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './Button.css';
+
+const Button = ({id, type, className, clickHandler, isDisabled, children}) => {
+    return (
+        <button
+            id={id}
+            type={type}
+            className={className}
+            onClick={clickHandler}
+            disabled={isDisabled}
+        >
+            {children}
+        </button>
+    );
+};
```

Comment:
Dit component is nu een doorgeefluik. Alle properties die binnenkomen worden direct doorgezet op één component. Je kan dus net zo goed direct het html button element gebruiken